Specs/Scene/ShadowMapSpec.js

```diff
@@ -50,8 +50,12 @@ describe(
     const floorHeight = -1.0;
 
```

Comment:
Also, I notice that the spec `set normalOffset` seems to have a `SPEC HAS NO EXPECTATIONS` when I run it in spec runner, is that expected?